1.1/spec.md

```diff
@@ -425,7 +425,7 @@ The following properties should be present:
   * syntactically, must be a UNIX-like pathname or http URL (no Windows pathnames)
   * may be relative
   * cannot be resolved to the actual file in general (e.g., if the hOCR file
-    becomes separated from the image fiels)
+    becomes separated from the image files)
```

Comment:
According to the context, I think it should be `image file` and not `image files` .

Reply:
It turns out I fixed this typo more than 5 years ago!
 
https://github.com/kba/hocr-spec/commit/5552893beeebbd218a81a49de72a729fe64626f2

but only in 1.2.

Reply:
Thank you for the review and your hint. I fixed that now.